webpack/scenes/ContentViews/Details/ComponentContentViews/ContentViewComponents.js

```diff
@@ -0,0 +1,114 @@
+import React, { useState, useEffect } from 'react';
+import { useSelector } from 'react-redux';
+import {
+  Bullseye,
+} from '@patternfly/react-core';
+import { Link } from 'react-router-dom';
+import { TableVariant, fitContent, TableText } from '@patternfly/react-table';
+import { STATUS } from 'foremanReact/constants';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { urlBuilder } from 'foremanReact/common/urlHelpers';
+import PropTypes from 'prop-types';
+
+import TableWrapper from '../../../../components/Table/TableWrapper';
+import onSelect from '../../../../components/Table/helpers';
+import {
+  selectCVComponents,
+  selectCVComponentsStatus,
+  selectCVComponentsError,
+} from '../ContentViewDetailSelectors';
+import { getContentViewComponents } from '../ContentViewDetailActions';
+import AddedStatusLabel from '../../../../components/AddedStatusLabel';
+import ComponentVersion from './ComponentVersion';
+import ComponentEnvironments from './ComponentEnvironments';
+import ContentViewIcon from '../../components/ContentViewIcon';
+
+const ContentViewComponents = ({ cvId }) => {
+  const response = useSelector(state => selectCVComponents(state, cvId));
+  const status = useSelector(state => selectCVComponentsStatus(state, cvId));
+  const error = useSelector(state => selectCVComponentsError(state, cvId));
+
+  const [rows, setRows] = useState([]);
+  const [metadata, setMetadata] = useState({});
+  const [searchQuery, updateSearchQuery] = useState('');
+  const columnHeaders = [
+    { title: __('Type'), transforms: [fitContent] },
+    __('Name'),
+    __('Version'),
+    __('Environments'),
+    __('Repositories'),
+    { title: __('Status') },
+    __('Description'),
+  ];
+  const loading = status === STATUS.PENDING;
+
+  const buildRows = (results) => {
+    const newRows = [];
+    results.forEach((componentCV) => {
+      const { content_view: cv, content_view_version: cvVersion } = componentCV;
+      const { environments, repositories } = cvVersion;
+      const {
+        id,
+        name,
+        description,
+      } = cv;
+
+      const cells = [
+        { title: <Bullseye><ContentViewIcon composite={false} /></Bullseye> },
```

Comment:
The tooltip says "single"; shouldn't it say "component"?

Reply:
I have a PR here to update the icon for Composite and Component..I'll add this change in the PR: https://github.com/Katello/katello/pull/9298

Reply:
Updated in the PR above.